snuba/query/processors/physical/clickhouse_settings_override.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, MutableMapping
```

Comment:
I know this is a very simple query processor so it doesn't feel like it needs a test but it should have a test (event if it is simple)

Reply:
Yeah, I was debating whether this needs a test, but I'll add a test for the sake of it